Toggl.Droid/Views/FloatingActionButtonAnimation.cs

```diff
@@ -0,0 +1,27 @@
+using Android.Views.Animations;
+
+namespace Toggl.Droid.Views
+{
+    public class FloatingActionButtonAnimation : Animation
+    {
+        private const float animationStartDelayThreshold = 0.05f;
+        private const float fullCircle = 360;
+        private float targetAngle;
+
+        private AnimatedFloatingActionButton animatedFloatingActionButton;
+
+        public FloatingActionButtonAnimation(AnimatedFloatingActionButton animatedFloatingActionButton)
+        {
+            targetAngle = fullCircle + animatedFloatingActionButton.AngleStartPoint;
+            this.animatedFloatingActionButton = animatedFloatingActionButton;
+        }
+
+        protected override void ApplyTransformation(float interpolatedTime, Transformation t)
+        {
+            if (interpolatedTime < animationStartDelayThreshold) return;
```

Comment:
Do not put blocks in the same line as flow-control statements.
```suggestion
            if (interpolatedTime < animationStartDelayThreshold) 
                return;
```